### PR TITLE
JCES-1756: open source checklist

### PR DESCRIPTION
**Pull request by dagguh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+<dependency>
+    <groupId>com.atlassian.db.replica</groupId>
+    <artifactId>db-replica</artifactId>
+    <version>0.1.17</version>
+</dependency>
+```
+
+## Documentation
+
+TODO: make Javadocs great
```
Comment (highvoltag3) [inline README.md:50]: @dagguh Would it be possible to have some minimum documentation before the checklist PR get's merged?

**Review by highvoltag3 — DISMISSED** [review]
Added a minor comment on the documentation section.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+```
+
+## Installation
+
+Maven:
+```xml
+<dependency>
+    <groupId>com.atlassian.db.replica</groupId>
+    <artifactId>db-replica</artifactId>
+    <version>0.1.17</version>
```
Comment (wyrzyk) [inline README.md:44]: 0.1.19 😉

File: README.md (hunk 1 above)
Comment (wyrzyk) [reply README.md:50]: @dagguh I think we can link dual-connection-states.md here.

Comment (dagguh) [reply README.md]: @highvoltag3 roger that, done!
@wyrzyk yes indeed ❤️

File: README.md (hunk 2 above)
Comment (dagguh) [reply README.md:44]: I'll think how to make it better in another PR

File: src/main/java/com/atlassian/db/replica/api/circuitbreaker/BreakerState.java [hunk 3]
```diff
@@ -1,5 +1,8 @@
 package com.atlassian.db.replica.api.circuitbreaker;
 
+/**
+ * States of <a href="https://www.martinfowler.com/bliki/CircuitBreaker.html">circuit breakers</a>.
```
Comment (wyrzyk) [inline src/main/java/com/atlassian/db/replica/api/circuitbreaker/BreakerState.java:4]: 👍

**Review by wyrzyk — APPROVED** [review]

**Pull request merged**